src/styles/_catalog.scss

```diff
@@ -16,7 +16,7 @@
     margin-bottom: 48px;
     cursor: pointer;
     transition-duration: 300ms;
-    transition-property: all;
+    transition-property: size color;
```

Comment:
You can check on dev tools how it works
<img width="285" alt="Screenshot 2022-07-20 at 17 41 59" src="https://user-images.githubusercontent.com/47606379/180010947-0b1b0e35-0561-4d96-830a-97440f5c0670.png">